### PR TITLE
edited responses document

### PR DESCRIPTION
**Pull request by andrewkern**
took as pass through the responses doc

### CODE REVIEW

**Comment by petrelharp** [comment]
Looks good generally! I've made a pass through myself; will comment on my commit.

File: response-to-reviewers.md [hunk 1]
```diff
@@ -10,6 +10,13 @@ our level-best to incorporate all of the changes suggested in the reviews.
 While that is so, in a few places we have decided that suggested changes were beyond
 the scope of the current manuscript and leave them for future work.  
 
+An additional change that has occurred since the first submission of the manuscript
+is that the Gutenkunst 2009 model hadn't been through QC, and had a bug.
+This has been corrected, resulting in mostly cosmetic changes to the figures and text
+(inferred population histories are slightly better than before; see Figure 4);
+but we have also explained the situation,
+as an example of the utility of QC and the resource more generally.
+
```
Comment (petrelharp) [inline response-to-reviewers.md:19]: thought we should mention this

File: response-to-reviewers.md [hunk 2]
```diff
@@ -92,35 +99,28 @@ and fully agree that the language in the manuscript needs to better emphasize th
...
-The catalog also contains numerous published demographic models for the first three species.
-These available models and parameters are meant to represent the field's current understanding,
-and we intend for this resource to evolve as new results become available."
+"For each species, the catalog contains curated information on our current understanding of
+the physical organization of its genome, inferred genetic maps,
+population-level parameters (e.g., mutation rate and generation time estimates),
+and published demographic models.
+These models and parameters are meant to represent the field's current understanding,
+and we intend for this resource to evolve as new results become available, and
+other existing models are added to stdpopsim by the community."
```
Comment (petrelharp) [inline response-to-reviewers.md:108]: Throughout, I've checked what we actually say in the manuscript and updated this.

File: response-to-reviewers.md [hunk 3]
```diff
@@ -285,11 +287,16 @@ See also our response to related comments made by Reviewers #1 and #2.
...
-but do reflect general trends observed in inference. The "zigzag" model could definitely be one of these, and indeed was initially implemented as a generic. However the reason we settled on the zigzag model is being defined as a human model is that its
-effective population size values are taken from (or at least inspired by) values inferred from human genomes. To see our internal discussion of these issues from September 2019 please see this [closed issue](https://github.com/popsim-consortium/stdpopsim/issues/106).
+We considered both of these suggestions, and here's why we've left it the way it is.
+Most of the columns of the table don't apply to generic models,
+so it seems strange to include them there.
+The "zigzag" model could definitely be a generic model,
+and indeed was initially implemented as such.
+However, the reason we settled on the zigzag model is being defined as a human model is that its
+effective population size values are taken from (or at least inspired by) values inferred from human genomes: as implemented, it is not as "generic" as it might seem.
+
```
Comment (petrelharp) [inline response-to-reviewers.md:298]: The issue didn't seem all that helpful, really.

**Comment by petrelharp** [comment]
Most of my changes were to update the quotes from the text.

**Review by petrelharp — APPROVED** [review]
LGTM! (esp with my changes =) I'll leave this open in case anyone else wants a look.

File: response-to-reviewers.md (hunk 3 above)
Comment (andrewkern) [reply response-to-reviewers.md:298]: happy to delete it, but man it's sweet to be able to point to github and say "look reviewers we talked about this!"

File: response-to-reviewers.md (hunk 1 above)
Comment (andrewkern) [reply response-to-reviewers.md:19]: i like it

File: response-to-reviewers.md (hunk 3 above)
Comment (petrelharp) [reply response-to-reviewers.md:298]: good point; put this back in

**Review by jeromekelleher — APPROVED** [review]

**Pull request merged**